calculator_test.go

```diff
@@ -2,23 +2,49 @@ package calculator_test
 
 import (
 	"calculator"
+	"fmt"
 	"testing"
 )
 
+type testcase struct {
```

Comment:
This is perfectly correct, but it's common in tests to not even give the test case type a name—after all, we never need to refer to it! We can simply supply an anonymous struct type:

```go
tcs := []struct{
	a, b float64
	want float64
}{
	{ a: 5, b: 2, want: 10 },
	...
}
```

Reply:
Thanks - noted. I think I was anticipating using it in other tests - which I haven't used yet and is pretty much against the spirit of TDD. 

Question - if I were to use the same struct in multiple tests would you declare as a standalone type, or would you do the same, e.g. inline in each test ?